.babelrc

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Hey Parvesh, I intentionally deleted the babelrc file as it's now using ababel.config.js file instead. This has the same configuration so just need to figure out why the app is not reading it

Reply:
yes, we can look into that later, why `babel.config.js` is not working. but let's unblock the develop for now.